extensions/wikia/GlobalNavigation/scripts/GoogleSearchAPI.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * We AB test for wikia search (both within and all wikias) in Oasis with Google custom search result.
+ * window.wgGoogleSearchTest: a global variable set true in Optimizely for 50% of users who visit ja wikia.
+ * window.wgGoogleSearchParam: Search ID set in Optimizely.
+ */
+$(function () {
+	if ( window.wgGoogleSearchTest ) {
+		var searchForm = document.getElementById('searchForm'),
```

Comment:
Is there any reason why you are using both vanilla JS and jQuery for getting DOM elements in a single JS file?